recipes/lemon/bld.bat

```diff
@@ -0,0 +1,18 @@
+mkdir build
+cd build
+
+cmake ^
+    -G "%CMAKE_GENERATOR%" ^
+    -DCMAKE_PREFIX_PATH=%LIBRARY_PREFIX% ^
+    -DCMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% ^
+    ..
+
+cmake --build . --config Release
+
+cmake --build . --config Release --target error_test
+cmake --build . --config Release --target bfs_test
+
+ctest -R error_test
+ctest -R bfs_test
```

Comment:
The tests are not built by default, they are excluded from the ``all`` target by default. One has to enable the ``Maintainer`` build type to have them built by default, but it's not clear if this has other consequences and preferred to err on safe side.

(Also, one test does not compile - even on linux).